src/CommunityToolkit/Xamarin.CommunityToolkit.Sample/Pages/Effects/LifeCycleEffectPage.xaml.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.CommunityToolkit.Sample.Pages.Effects
+{
+	public partial class LifeCycleEffectPage
+	{
+		public LifeCycleEffectPage()
+		{
+			InitializeComponent();
+		}
+
+		void LifeCycleEffect_Loaded(object sender, EventArgs e)
+		{
+			if (sender is Button)
```

Comment:
Maybe we can include a Label in the UI that is updated every time a loaded/unloaded event is invoked (Whoever tries the sample will see everything in debug and release mode).

Reply:
Nice suggestion!